### ISSUE TITLE
unable to add/authenticate device to Ubuntu One account

### ISSUE THREAD

**Issue by obilodeau** [tomboy-notes/tomdroid#91]
Hi, I just installed the Tomdroid Notes app on my Samsung Galaxy S3 with Android 4.1.1 and I'm unable to authenticate the device with Ubuntu One in order to sync notes between my Ubuntu pc and my phone.

In Tomdroid, under Synchronization I have chosen "Tomboy Web" as the Service and I have "https://one.ubuntu.com/notes" as the Server. When I select Authenticate, my browser opens and I log into Ubuntu One, on the "Confirm Computer Access" webpage I input a name for the device and select "Add This Computer" and get the "Something has gone wrong" page. What's going on? Thank you.

---

Imported from Launchpad using lp2gh.
- date created: 2013-01-02T22:33:36Z
- owner: logicalcombine
- the launchpad url was https://bugs.launchpad.net/bugs/1095464

**Comment by obilodeau** [comment]
(by logicalcombine)
I tried adding the decice again this morning and it was successful.  Issue resolved.

**Comment by obilodeau** [comment]
(by obilodeau)
Thanks for letting us know it worked in the end.

**Comment by obilodeau** [comment]
(by jmozer)
Same bug with me on Android 4.0, since yesterday...

**Comment by obilodeau** [comment]
(by sirniemi)
When I first time authenticated (12.1.) device with older Google Play version it apparently did work but I did not sync notes and for some reason decided to remove device authentication Ubuntu One account page to do it again. Since then I haven't been able to authenticate again with older and this newer 0.7.2 version. Using Nexus 7, Android 4.2.1. Tried clearing data from Tomdroid and Chrome but no effect.
